src/SharedAssemblyInfo.cs

```diff
@@ -1,7 +1,10 @@
+using System;
 using System.Reflection;
+using System.Runtime.CompilerServices;
+
```

Comment:
At one point, I added a TypeForewardedToAttribute to try to solve a typing issue with JSON.NET and ICloneable. But updating to the latest JSON.NET solved that problem.